src/test/java/se/sundsvall/byggrarchiver/service/scheduler/BatchSchedulerShedlockTest.java

```diff
@@ -0,0 +1,98 @@
+package se.sundsvall.byggrarchiver.service.scheduler;
+
+import static java.time.Clock.systemUTC;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.within;
+import static org.awaitility.Awaitility.await;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.test.context.ActiveProfiles;
+import se.sundsvall.byggrarchiver.service.ByggrArchiverService;
+
+@SpringBootTest(properties = {
+	"scheduler.cron.expression=* * * * * *", // Setup to execute every second
+	"spring.flyway.enabled=true",
+	"spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver",
```

Comment:
This is already set in the application-junit.yaml, right?

Reply:
oui.. lazy copy pasta